specification/widget/data-plane/Widget/tspconfig.yaml

```
@@ -0,0 +1,46 @@
+parameters:
+  "service-dir":
+    default: "sdk/widget"
+  "dependencies":
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emit-lro-options: "none"
+    emitter-output-dir: "{project-root}/../.."
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/widgets.json"
```

```suggestion
  "@azure-tools/typespec-autorest":
    emit-lro-options: "none"
    emitter-output-dir: "{project-root}"
    output-file: "{version-status}/{version}/widgets.json"
```

@qiaozha, @markcowl: Since `tspconfig.yaml` is now a sibling of the `preview/stable` folders, should we simplify the autorest config like this, or is it important to either:

1. Keep setting `azure-resource-provider-folder`, even though we no longer need it for output-file?
2. Include `{service-name}` in `output-file`?  Without this, I think it would be possible for the "service folder" to use a different name than the "default namespace" in TSP.

---

I like the simplified way.